meta-buildpacks/java/buildpack.toml

```diff
@@ -2,7 +2,7 @@ api = "0.4"
 
```

Comment:
Unrelated to this PR, but I noticed that the meta-buildpacks here are still using `api = "0.4"`. I can't remember what effect (if any) changing buildpack API has on meta-buildpacks, but we should probably update these versions at some point in the future.

Reply:
I'll do this in a separate PR when I bring this repo up to date for the new "rust only" world later. :)